staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/admission.go

```diff
@@ -108,6 +111,10 @@ func (c *celAdmissionPlugin) SetDrainedNotification(stopCh <-chan struct{}) {
 	c.stopCh = stopCh
 }
 
+func (c *celAdmissionPlugin) SetAuthorizer(authorizer authorizer.Authorizer) {
+	c.authorizer = authorizer
```

Comment:
do we need to check authorizer is non-nil in `ValidateInitialization()`?

Reply:
Added, thx!